### DISCUSSION TITLE
Is it possible to see any "errors" happening on the ebus?

### DISCUSSION THREAD

**Discussion by Commifreak** [john30/ebusd#612] [Q&A]
I dont have any ebus HW connected (yet).

I recently had an VR71 communication error on my multiMatic, which causes the heater to go crazy and sends permanent heat to the "underfloor heating" (dont know the exact english term). This happened two times (3/4 year between it).

I reported that to Vaillant and they answered it could be a bad ebus wiring. And indeed, they said, "I see some random communication erros here and there happening, especially to the outdoor temp sensor" (never had an issue with that, though).

My question related to ebusd: Is ebusd able to show any those (communication) errors and what exactly went wrong?

Thanks in advance :)

**Comment by john30** [comment]
with an [adapter 3](https://adapter.ebusd.eu/index.en.html), the PIC UART reports framing errors to ebusd and can be adjusted wrt the arbitration timing

**Reply by Commifreak** [reply]
Thanks.

But a adjustment does not solve a main wiring problem, which seem to be the case here. But without any error reporting I cant tell where to start, so I will see, when my order arrives :)
